scratch-debugger/debug.sh

```diff
@@ -146,19 +151,13 @@ spec:
         - "${INSTALLCMD}"
       # Mount the node FS for direct access to docker.
       volumeMounts:
-        - name: rootfs
-          mountPath: /mnt/rootfs
-          readOnly: true
-        - name: rootfs-run
-          mountPath: /mnt/rootfs/var/run
+        - name: docker-sock
+          mountPath: /mnt/docker.sock
           readOnly: true
```

Comment:
The container can talk to the socket even if readOnly is set to true?

Reply:
yes.